components/Loading.tsx

```diff
@@ -21,7 +21,7 @@ const Loading: FC<LoadingProps> = (props: LoadingProps) => {
     }, 1000)
```

Comment:
you might want to return a callback from useEffect which will clear the interval. Technically it shouldn't be required because you've specified it will just run once but this ensures if react re-runs it it will destroy the previous interval first